### PR TITLE
Sgds 176

### PR DESCRIPTION
**Pull request by haphizi**
Changes:
paginationExtended component to make storybook props togglelable
WIP: test, directionVariant: 'icon' | 'icon-text' |'text',
dropdownMenu component offset default is fixed to 10px
WIP: need to add back popperconfig code snippet into story

### CODE REVIEW

**Review by clukhei — APPROVED** [review]
my reviews can be rectified in the next branch you work on :)

File: stories/Dropdown/Dropdown.stories.mdx [hunk 1]
```diff
@@ -19,7 +19,15 @@ import { Dropdown, Row, Col } from '../../src/components/index';
         type: { summary: "true | 'outside' | 'inside' | false" },
         defaultValue: { summary: 'true' },
       },
-      control: { disable: true },
+      control: {
```
[the remark this answers is not in the document]
Comment (haphizi) [reply stories/Dropdown/Dropdown.stories.mdx:22]: previously i thought it was not needed, i will remove the disabling.

File: stories/Dropdown/Dropdown.stories.mdx [hunk 2]
```diff
@@ -77,21 +86,10 @@ import { Dropdown, Row, Col } from '../../src/components/index';
 
 Toggle contextual overlays for displaying lists of links
 
-export const popperConfig = {
```
[the remark this answers is not in the document]
Comment (haphizi) [reply stories/Dropdown/Dropdown.stories.mdx:80]: noted

File: stories/Pagination/Pagination.stories.mdx [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
```
[the remark this answers is not in the document]
Comment (haphizi) [reply stories/Pagination/Pagination.stories.mdx:1]: noted

**Pull request merged**